docs/_articles/writing-style.md

```diff
@@ -0,0 +1,15 @@
+---
+title: UI text writing style
+---
+
+The rules are:
+
+- Capitalize first letter of everything - from single words to sentences.
+- Finish with a period only if it's a full sentence - e.g. not if it's a heading, short instruction, etc.
+- Always make punctuation part of the translated string (so that RTL languages don't start with a period, for example).
```

Comment:
let's remember to update this documentation when we switch to string identifiers in the UI code

Reply:
I've added a TODO_NOTE :)